x-pack/platform/plugins/shared/alerting/server/usage/task_state.ts

```diff
@@ -64,6 +64,8 @@ const stateSchemaV1 = schema.object({
   count_rules_snoozed: schema.number(),
   count_rules_muted: schema.number(),
   count_rules_with_muted_alerts: schema.number(),
+  count_rules_with_linked_dashboards: schema.number(),
+  count_rules_with_investigation_guide: schema.number(),
```

Comment:
Please, follow the pattern mentioned in the beginning of this file, add version 3:
```
/**
 * WARNING: Do not modify the existing versioned schema(s) below, instead define a new version (ex: 2, 3, 4).
 * This is required to support zero-downtime upgrades and rollbacks. See https://github.com/elastic/kibana/issues/155764.
 *
 * As you add a new schema version, don't forget to change latestTaskStateSchema variable to reference the latest schema.
 * For example, changing stateSchemaByVersion[1].schema to stateSchemaByVersion[2].schema.
 */
```

Reply:
thanks @guskovaue, I missed that, updated in 5632e604af466fa8e15c6832fa47c902b1c18d13